airtable/plugin.py

```diff
@@ -37,7 +37,13 @@ def airtable_url_to_ids(url: str) -> Tuple[AirtableBaseID, AirtableTableID]:
         raise ValueError("Invalid Airtable URL")
 
     base_id = match.group(1)
-    table_id = match.group(2) if match.group(2) else None
+    # When loading the Airtable using the URL that only has the base id (e.g. https://airtable.com/apptpbyiHjjfPecFw)
+    # Airtable automatically drops you into the first table of the base and we don't expect the table_id to be an empty string
+    # if the user copies the URL from the browser.
+    if not match.group(2):
+        raise ValueError("Invalid Airtable table ID")
```

Comment:
Can we have a better error message  that says something about the url? This gets propagated to code gen, so useful to have it be a good message